heron/scheduler-core/src/java/org/apache/heron/scheduler/RuntimeManagerRunner.java

```diff
@@ -478,4 +551,16 @@ private static boolean parallelismChangeDetected(PackingPlans.PackingPlan curren
     }
     return false;
   }
+
+  private static boolean changeDetected(PackingPlans.PackingPlan currentProtoPlan,
```

Comment:
What's the difference between `changeDetected ` and `parallelismChangeDetected `?